src/pandapipes/component_models/heat_consumer_component.py

```diff
@@ -224,5 +246,6 @@ def extract_results(cls, net, options, branch_results, mode):
         """
         required_results_hyd, required_results_ht = standard_branch_wo_internals_result_lookup(net)
 
-        extract_branch_results_without_internals(net, branch_results, required_results_hyd,
-                                                 required_results_ht, cls.table_name(), mode)
+        extract_branch_results_without_internals(
+            net, branch_results, required_results_hyd, required_results_ht, cls.table_name(), mode
+        )
```

Comment:
Do we need to extract qext here? Should be rather important for heat consumers with return temperature control?